.github/workflows/next-build.yml

```diff
@@ -7,7 +7,7 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-name: CI
+name: Next build
```

Comment:
Next DockerimagebIs it convention? I mean using `build` since it may be understood like only build without storing images. In DWO we have it `Next Dockerimage` which may be a bit cleaner.

Reply:
Would "Next container image" be better then? Docker is a competitor, and a specific implementation (which isn't podman). :)

Reply:
so I guess we could go with whatever project owners prefer, as this is simply a description (we hadn't come up with conventions for it).

Reply:
+1 for "Next container image"

Reply:
> Would "Next container image" be better then? Docker is a competitor, and a specific implementation (which isn't podman). :)

good catch. Next container image works for me

Reply:
> whatever project owners prefer

Nope, we should pick one convention and use that everywhere. What value is there in being inconsistent when the whole rationale to rename files here is to be consistent across the repos? :D